python/setup.cfg

```diff
@@ -16,7 +16,6 @@ classifiers =
     Programming Language :: C
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
```

Comment:
Should add 3.11 here

Reply:
Good spot!

Reply:
Just checked and it was already there - just snipped off this diff.

Reply:
Ah, my bad